UploadFile/Codeunit/Cod50410.BCPUploadFile.al

```diff
@@ -0,0 +1,57 @@
+codeunit 50410 "BCP UploadFile"
+{
+
+    procedure UploadFile(TableID: Integer; RecordNo: Code[20])
+
+    var
+        uploadTable: Record "BCP Upload";
+        FileInStream: InStream;
+        tempfilename: text;
+        DialogTitleMsg: Label 'Please select a File...';
+
+    begin
+        if UploadIntoStream(DialogTitleMsg, '', 'All Files (*.*)|*.*', tempfilename, FileInStream) then begin
```

Comment:
I suggest you write something like this
```
if not UploadIntoStream(DialogTitleMsg, '', 'All Files (*.*)|*.*', tempfilename, FileInStream) then
  exit;
```

if not exit statement will reduce number of nesting

Reply:
That is noted